Cargo.toml

```diff
@@ -240,7 +240,7 @@ homepage = "https://aptoslabs.com"
 license = "Apache-2.0"
 publish = false
 repository = "https://github.com/aptos-labs/aptos-core"
-rust-version = "1.64"
+rust-version = "1.70.0"
```

Comment:
I wonder, do we want to set patch version explicitly, usually it's pretty safe to leave that unspecified right? This way we pick up security fixes for free.

Reply:
This is just a minimum version (according to the doc quoted in the PR description) so it should be fine. IIUC 1.70.0 should be equivalent to 1.70. I'm fine to change it as well if you feel it's different

Reply:
Let me just change it -- the example doesn't have a patch version, so maybe it's not a good idea to include one :)